rust/chains/hyperlane-ethereum/src/trait_builder.rs

```diff
@@ -104,6 +103,34 @@ pub trait BuildableWithProvider {
                 self.wrap_with_metrics(quorum_provider, locator, signer, middleware_metrics)
                     .await?
             }
+            ConnectionConf::HttpFallback { urls } => {
+                let rpc_metrics = rpc_metrics.map(|f| f());
+                let mut builder = FallbackProvider::builder();
+                let http_client = Client::builder()
+                    .timeout(HTTP_CLIENT_TIMEOUT)
```

Comment:
Just wanna flag that afaik we we haven't actually experienced many timeouts with our requests, but if for some reason the first provider has issues that result in the timeout consistently getting hit, we'll consistently take 1 min per request

Generally I don't think we should make any logic changes yet to deal with consistently failing primary providers, but I think we should set up alerts on the rate of this metric so that we can be aware if we're having a lot of failures:
```
hyperlane_request_failure_count{method!~"(eth_estimateGas|eth_sendRawTransaction|eth_sendTransaction)"}
```